syntax = "proto3";


option java_multiple_files = true;
option java_package = "edu.ucsc.edgelab.cryptopg";
option java_outer_classname = "CrptoPG";
option objc_class_prefix = "HLW";

package cryptopg;

service CryptoPG {
    rpc GetPKCS8PublicKey (Dummy) returns (PublicKey) ;
}

message Dummy {

}
message PublicKey {
    bytes pkcs8PublicKey = 1;
}